lib/child_process.js

```diff
@@ -468,14 +468,19 @@ function normalizeSpawnArguments(file, args, options) {
 
   if (options.shell) {
     const command = [file].concat(args).join(' ');
-
+    // Set the shell, switches, and commands.
     if (process.platform === 'win32') {
       if (typeof options.shell === 'string')
         file = options.shell;
       else
         file = process.env.comspec || 'cmd.exe';
-      args = ['/d', '/s', '/c', `"${command}"`];
-      options.windowsVerbatimArguments = true;
+      // '/d /s /c' is used only for cmd.exe.
+      if (/^(?:.*\\)?cmd(?:\.exe)?$/i.test(file)) {
```

Comment:
A late comment, I think that `\b` might be a simpler test for the prefix:
```
if (/\bcmd(\.exe)?$/i.test(file)) 
```
also IMHO no need to `?:` the groups since we just `.test`

Reply:
P.S. I would call my comment a "nit" (that is a non blocking style change)